stl/inc/thread

```diff
@@ -364,6 +364,10 @@ public:
         return thread::hardware_concurrency();
     }
 
+    _NODISCARD native_handle_type native_handle() {

```

Comment:
Could we please move this up between `get_id` and `get_stop_source` to agree with the order of declarations in [[thread.jthread.class.general]](http://eel.is/c++draft/thread.jthread.class.general)?

Reply:
Good callout. Moved in both `thread` and `jthread`.